### DISCUSSION TITLE
Lineage support for GlueDyanamicFrame

### DISCUSSION THREAD

**Discussion by rushabh1995** [AbsaOSS/spline-spark-agent#787] [Ideas]
Hi guys,
I am trying to extract lineage from Glue job using spline JAR **(spark-3.3-spline-agent-bundle_2.12 JAR 2.0.0)** which works only for spark DataFrame but not glue dynamic frame.
Is there any functionality provided in spline JAR or anything which will be helpful to extract the lineage of Glue DynamicFrame.
Currently, we are using Glue 4.0 version for our UseCase.

Can you provide Glue dyanamicFrame functionality support in your upcoming release??

**Comment by wajda** [comment]
See #781
